frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -52,9 +51,11 @@ const StartRunModal: React.FC<StartRunModalProps> = ({ onSubmit, onCancel }) =>
               }}
             >
               LAB method
-            </Button>
-            . Before creating a run, a taxonomy is needed and a teacher and judge model must be
-            configured.{' '}
+            </Button>{' '}
+            with the InstructLab pipeline. To create a LAB-tuning run, you must have a taxonomy
+            stored in a git repository, and a configured teacher and judge model.{' '}
```

Comment:
I don't think space is needed here.

```suggestion
            stored in a git repository, and a configured teacher and judge model.
```